common/app/views/support/cleaner/VideoEmbedCleaner.scala

```diff
@@ -57,22 +57,21 @@ case class VideoEmbedCleaner(article: Article) extends HtmlCleaner {
 
         val video = findVideoApiElement(mediaId)
 
+        // remove all sources as we add them later in a specific order
+        // similar to video.scala.html template
+        // see `common.app.model.EncodingOrdering` for order
         element.getElementsByTag("source").remove()
 
-        val sourceHTML: String = getVideoAssets(mediaId).map { videoAsset =>
-          videoAsset.encoding.map { encoding =>
-            s"""<source src="${encoding.url}" type="${encoding.format}"></source>"""
-          }.getOrElse("")
-        }.mkString("")
-
-        element.append(sourceHTML)
-
         // add the poster url
         video.map(_.images).flatMap(Item640.bestFor).map(_.toString()).foreach { url =>
           element.attr("poster", url)
         }
 
-        findVideoApiElement(mediaId).foreach { videoElement =>
+        video.foreach { videoElement =>
+          videoElement.videos.encodings.map { encoding => {
```

Comment:
What about adding a test to the `VideoMedia` model for instance to verify the expected encoding ordering? That would make it obvious that the ordering is actually set this way on purpose and modifying it by accident would break the test.